ova/generate_ova.sh

```diff
@@ -21,39 +21,23 @@ scriptpath=$(
 
 OUTPUT_DIR="${scriptpath}/output"
 CHECKSUM_DIR="${scriptpath}/checksum"
-HAVE_VERSION=false
-HAVE_OPENDISTRO_VERSION=false
-HAVE_ELK_VERSION=false
-
-# Get the currents production branches
-BRANCHDOC=$(curl -s https://documentation.wazuh.com/current/index.html | grep -o "m/[0-9\.]\+/i" | head -n1)
-BRANCHDOC="${BRANCHDOC:2:3}"
-BRANCH=${BRANCHDOC}
-export BRANCHDOC
-export BRANCH
-
-WAZUH_VERSION=""
-OPENDISTRO_VERSION=""
-ELK_VERSION=""
+HAVE_S3REPO="no"
+
+# wazuh-packages branch/tag of unattended installer script to use
+PACKAGES_BRANCH="4.2"
```

Comment:
By default to `""`

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/832/commits/9d4132b545b62ad4399d421d3e37db7dc3a73ea2.

Removed, when using the local unattended this variable is no longer necessary since the version is taken from the unattended script itself